tasks/update-granules-metadata/schemas/config.json

```diff
@@ -0,0 +1,72 @@
+{
+  "title": "UpdateGranulesMetadataConfig",
+  "description": "Describes the config used by the update-granules-metadata task",
+  "type": "object",
+  "required": [
+    "bucket",
+    "distribution_endpoint",
+    "collection",
```

Comment:
I'm not seeing where config.collection is used in the task, so I think that can be removed from the config schema?

Reply:
Yeah, that's true. I'll remove it